package.json

```diff
@@ -45,9 +45,7 @@
   },
   "dependencies": {
     "@babel/core": "^7.6.4",
-    "@babel/parser": "^7.6.4",
     "@babel/preset-env": "^7.6.3",
-    "@babel/traverse": "^7.6.3",
```

Comment:
It’s a shame that we don’t gain anything from dropping these since they’re deps of other Babel modules! Still, it brings us closer to dropping Babel altogether which is cool

Reply:
Yea for sure, the benefit here is definitely more speed & stability (a lot less can go wrong when you're just scanning and not also parsing it into a valid AST). But you're right that the total dep count probably hasn't dropped by that much. 🤷‍♂